setup.py

```diff
@@ -6,10 +6,10 @@
 
 from setuptools import find_packages, setup
 
-with open("README.md") as readme_file:
+with Path.open(Path("README.md")) as readme_file:
     readme = readme_file.read()
 
-with open("HISTORY.md") as history_file:
+with Path.open(Path("HISTORY.md")) as history_file:
     history = history_file.read()
```

Comment:
Are these Path.open changes suggestions from ruff?

Reply:
ruff suggests using `Path.open()` which generates an error with `str`. I think it should be Path("HISTORY.md").open()